api/cpp/CMakeLists.txt

```diff
@@ -153,6 +153,10 @@ endif()
 
 if (NOT SLINT_FEATURE_STD)
     list(APPEND features i-slint-core/libm i-slint-core/unsafe-single-threaded)
+
+    if (ESP_PLATFORM)
+        list(APPEND features "esp-backtrace/${IDF_TARGET}")
```

Comment:
I hope IDF_TARGET is always defined to a valid feature name of esp-backtrace when ESP_PLATFORM is defined

Reply:
Fingers crossed :)